src/ontology/cl-edit.owl

```diff
@@ -16452,17 +16453,18 @@ SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0002254 obo:CL_0002563
 
 # Class: obo:CL_0002255 (stromal cell of endometrium)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11331626") Annotation(oboInOwl:hasDbXref "PMID:30013421") Annotation(oboInOwl:hasDbXref "PMID:30309298") Annotation(oboInOwl:hasDbXref "Wiki:Decidualization&oldid=908981933#Endometrial_stromal_cells_(ESCs)") Annotation(oboInOwl:hasDbXref "https://doi.org/10.1038/s41591-020-1040-z") obo:IAO_0000115 obo:CL_0002255 "A stromal cell of the endometrium that is fibroblastic in appearance. During decidualization this cell may differentiate into a decidual stromal cell.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11331626") Annotation(oboInOwl:hasDbXref "PMID:30013421") Annotation(oboInOwl:hasDbXref "PMID:30309298") Annotation(oboInOwl:hasDbXref "PMID:31907034") Annotation(oboInOwl:hasDbXref "Wikipedia:Decidualization") Annotation(oboInOwl:hasDbXref "https://doi.org/10.1038/s41591-020-1040-z") obo:IAO_0000115 obo:CL_0002255 "A stromal cell of the endometrium, characterized by its fibroblast-like morphology, regenerative capacity, and ability to undergo decidualization. It differentiates into a decidual stromal cell during pregnancy, essential for embryo implantation and maintenance. This cell is involved in tissue proliferation, remodeling, and breakdown, responding to hormonal changes, particularly estrogen and progesterone.")
 AnnotationAssertion(terms:contributor obo:CL_0002255 <https://orcid.org/0000-0003-1980-3228>)
 AnnotationAssertion(oboInOwl:creation_date obo:CL_0002255 "2010-09-08T09:50:04Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasBroadSynonym obo:CL_0002255 "stromal cell of uterus")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0002255 "CALOHA:TS-1266")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0002255 "FMA:86490")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0002255 "endometrial stromal cell")
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30309298") oboInOwl:hasExactSynonym obo:CL_0002255 "endometrial stromal fibroblast")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31907034") oboInOwl:hasNarrowSynonym obo:CL_0002255 "decidualized endometrial stromal cell")
```

Comment:
I think it would be more appropriate a 'related synonym' rather than narrow, because it seems there is a differentiation relationship, not subclass. What do you think?

Reply:
decidualized endometrial stromal cells have been added as an exact synonym to decidual cell